pkg/kubectl/cmd/create.go

```diff
@@ -126,8 +126,8 @@ func printObjectSpecificMessage(obj runtime.Object, out io.Writer) {
 			An external load-balanced service was created.  On many platforms (e.g. Google Compute Engine),
```

Comment:
An ugly side effect of using back-quotes here is that all the whitespace at the front of lines is preserved.  Can we undent that?